src/Illuminate/Foundation/Bootstrap/ConfigureLogging.php

```diff
@@ -90,7 +90,8 @@ protected function configureDailyHandler(Application $app, Writer $log)
 	 */
 	protected function configureSyslogHandler(Application $app, Writer $log)
 	{
-		$log->useSyslog('laravel');
+		$name = $app->make('config')->get('app.log_syslog_name', 'laravel');
```

Comment:
Just `$app['config']` would be more consistent in here, rather than `app->make('config')`. Also, no point in that variable assignment first.